File: Python/1066.EvenOddPositiveandNegative.py
odd = 0
even = 0
positive = 0
negative = 0

for i in range(1, 6, 1):
    n = int(input())
    if(n % 2 == 0):
        even += 1
    else:
        odd += 1

    if(n > 0):
        positive += 1
    elif(n < 0):
        negative += 1

print(f"{even} valor(es) par(es)")
print(f"{odd} valor(es) impar(es)")
print(f"{positive} valor(es) positivo(s)")
print(f"{negative} valor(es) negativo(s)")

File: C/1066.EvenOddPositiveandNegative.c
#include <stdio.h>

int main(void){
    int n, odd=0, even=0, positive=0, negative=0;

    for(int i=0; i<5; i++){
        scanf("%d", &n);
        if(i%2==0){
            even++;
        }else{
            odd++;
        }
        if(n > 0){
            positive++;
        }else{
            negative++;
        }
    }

    printf("%d valor(es) par(es)\n%d valor(es) impar(es)\n%d valor(es) positivo(s)\n%d valor(es) negativo(s)\n", even, odd, positive, negative);

    return 0;
}